docs/tutorials/data_pipeline.md

```diff
@@ -146,28 +146,29 @@ For each operation, we list the related dict fields that are added/updated/remov
 
 ## Extend and use custom pipelines
 
-1. Write a new pipeline in any file, e.g., `my_pipeline.py`. It takes a dict as input and return a dict.
+1. Write a new pipeline e.g., `my_pipeline.py`. It takes a dict as input and returns a dict.
 
     ```python
+    import random
     from mmdet.datasets import PIPELINES
 
     @PIPELINES.register_module()
     class MyTransform:
+        def __init__(self, p=.5):
```

Comment:
suggest setting `p=0.5`, and add you can add docstring.
An example:
https://github.com/open-mmlab/mmdetection/blob/682f03db0c0516324ec4e1e5d97e1d93c0a2485a/mmdet/datasets/pipelines/transforms.py#L479-L482